### PR TITLE
fix: deploy doc

### PR DESCRIPTION
**Pull request by nic-chen**
remove useless description
fix config description.

### CODE REVIEW

File: docs/deploy.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ The bundled files are located in the root directory `/output/html`.
 
 ## Run
 
-1. According to your deploy environment, check the related configurations in `api/conf/conf.yaml`, modify those variables if needed.
+1. According to your deploy environment, check the related configurations in `output/conf/conf.yaml`, modify those variables if needed.
```
Comment (juzhiyuan) [inline docs/deploy.md:84]: Why? The output folder is generated only when we run the shell.

File: docs/deploy.md [hunk 2]
```diff
@@ -114,18 +113,15 @@ $ api/run.sh &
...
 ```json
-{
-  "etcd": {
-    "endpoints": "127.0.0.1:2379,127.0.0.1:3379"
-  }
-}
+  etcd:
+    endpoints:
+      - 127.0.0.1:2379
+      - 127.0.0.1:3379
```
Comment (juzhiyuan) [inline docs/deploy.md:124]: image

**Review by juzhiyuan — CHANGES_REQUESTED** [review]

File: docs/deploy.md (hunk 1 above)
Comment (nic-chen) [reply docs/deploy.md:84]: The binary program we run is in the `output` directory, so the config file it reads is `output/conf/conf.yaml`

Comment (juzhiyuan) [reply docs/deploy.md:84]: But we don't havr that directory in source codes, and users can choose not to run the binary file in that folder.

**Review by juzhiyuan — COMMENTED** [review]
Enmm

File: docs/deploy.md (hunk 1 above)
Comment (nic-chen) [reply docs/deploy.md:84]: Then he should change the config file in the directory he run. After  run `api/build.sh`, the directory of the binary file is `output`, changing `api/conf/conf.yaml` will not take effect.

Comment (nic-chen) [reply docs/deploy.md:84]: or what's your suggestion ? @juzhiyuan

Comment (juzhiyuan) [reply docs/deploy.md:84]: 1. Before building the binary file, we should update the config file under the `/api/conf/` directory, right?
2. After building successfully, we have the output directory.
3. When users want to update the config file without rebuilding the binary file, they must modify the file under the `/output` directory.

Comment (juzhiyuan) [reply docs/deploy.md:84]: According to your opinion, we should add some tips in the default config file, to let users know they can update the config only when the binary is generated.

Comment (moonming) [inline docs/deploy.md:84]: This is unacceptable, there can only be one configuration file

**Review by moonming — CHANGES_REQUESTED** [review]
This is unacceptable, there can only be one configuration file

**Comment by nic-chen** [comment]
@juzhiyuan @moonming 

Actually, users only need to care about `/output/conf/`, only developers need to care about `/api/conf/`

**Comment by juzhiyuan** [comment]
> @juzhiyuan @moonming 
> 
> Actually, users only need to care about `/output/conf/`, only developers need to care about `/api/conf/`
> 
> 

But users don't know

**Comment by membphis** [comment]
> @juzhiyuan @moonming
> 
> Actually, users only need to care about `/output/conf/`, only developers need to care about `/api/conf/`

that is a bad way. 

after build, we can generate `dag-to-lua` and `manager-api` in `/api`.

```
$ tree ./ -L 1
./
├── conf
├── dag-to-lua
├── Dockerfile
├── filter
├── go.mod
├── go.sum
├── internal
├── log
├── logs
├── manager-api
├── README.md
└── test
```

**Comment by nic-chen** [comment]
> > @juzhiyuan @moonming
> > Actually, users only need to care about `/output/conf/`, only developers need to care about `/api/conf/`
> 
> that is a bad way.
> 
> after build, we can generate `dag-to-lua` and `manager-api` in `/api`.
> 
> ```
> $ tree ./ -L 1
> ./
> ├── conf
> ├── dag-to-lua
> ├── Dockerfile
> ├── filter
> ├── go.mod
> ├── go.sum
> ├── internal
> ├── log
> ├── logs
> ├── manager-api
> ├── README.md
> └── test
> ```


We have other considerations, that is, users can directly copy the `output` directory to any location they want.

**Comment by juzhiyuan** [comment]
@membphis Yes, we couldn't build it under the api directory due to users may copy them and move them to another place.

**Comment by juzhiyuan** [comment]
Why not use a template-configuration? No matter use a standalone file or just added them to codes.

**Comment by membphis** [comment]
I think we need refractory the deploy doc.

Let us discuss in issue first.

**Comment by gxthrj** [comment]
I create a new issue to impove #715

**Comment by nic-chen** [comment]
so this pr should be closed now.

**Pull request closed without merging**